test/KitPidHelpers.cpp

```diff
@@ -58,7 +59,8 @@ void helpers::logKitProcesses(const std::string& testname)
             << " Spare Kits: " << getPidList(spareKitPids));
 }
 
-void helpers::waitForKitProcessCount(
+namespace {
+void waitForKitProcessCount(
```

Comment:
Please re-add a comment about -1 being a valid input here, that ignores counting that type of pid.

Also, please remove the comments showing the default argument values below. I just noticed those are still there but are incorrect, I must have forgotten to remove them after moving things around before submitting this file last time.

Reply:
Inlined the method instead, and removed quite some complexity there that we don't need; thanks.